packages/babel-plugin-transform-react-jsx-self/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "babel-plugin-transform-react-jsx-self",
+  "version": "6.0.14",
```

Comment:
Not sure if this matters but I would change this to be the current version so 6.0.14